package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -177,13 +178,43 @@ class H5MDReader(base.ReaderBase):
              'force': None}
     # Translate H5MD units (https://nongnu.org/h5md/modules/units.html) to MDAnalysis units.
     _unit_translation = {
+        'time': {
+            'ps': 'ps',
+            'fs': 'fs',
+            'ns': 'ns',
+            'second': 'second',
+            'sec': 'sec',
+            's': 's',
+            'AKMA': 'AKMA',
+            },
+        'length': {
+            'Angstrom': 'Angstrom',
+            'angstrom': 'Angstrom',
+            'A': 'Angstrom',
+            'nm': 'nm',
+            'pm': 'pm',
```

Comment:
Im pretty sure we dont support fm or pm

Reply:
https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/units.py#L195-L201 

I was reading off of this dictionary. Are these not the units mda uses to do unit conversion?

Reply:
Hah! We do. You’re right.